services/processor/native/repository/_Deployments/deploy.go

```diff
@@ -27,15 +28,16 @@ func getInfo(serviceName string) uint32 {
 	return processorType
 }
 
-func getCode(serviceName string) []byte {
-	code := _readCode(serviceName)
-	if len(code) == 0 {
-		panic("contract code not available")
-	}
-	return code
+func getCode(serviceName string, index uint32) []byte {
+	res := _readCode(serviceName, index)
+	return res
+}
+
+func getCodeParts(serviceName string) uint32 {
+	return _codeCounter(serviceName)
 }
 
-func deployService(serviceName string, processorType uint32, code []byte) {
+func deployService(serviceName string, processorType uint32, code ...[]byte) {
```

Comment:
did you verify with @OdedWx that this doesn't break protocol? the question here is whether a deployment transaction that was written against the deployment old contract (with no ellipsis) can be re-executed successfully against the new contract.

Reply:
It will be re-executed successfully.